pkg/diagnose/connections.go

```diff
@@ -20,7 +20,7 @@ package diagnose
 
 import (
 	"github.com/submariner-io/admiral/pkg/reporter"
-	"github.com/submariner-io/subctl/pkg/cluster"
+	"github.com/submariner-io/subctl/internal/cluster"
```

Comment:
`cluster.Info` is referenced in the public `Connections` function so if it's moved internal I think it would be problematic for users outside the repo. Perhaps we can move the _internal/cluster_  functions to _pkg/cluster_ instead.